### ISSUE TITLE
General queries not detected on terraform plan

### ISSUE THREAD

**Issue by LirycR** [Checkmarx/kics#6454]
Hello,

The following general terraform queries are not detected on terraform plan.
- [output_without_description](https://github.com/Checkmarx/kics/tree/master/assets/queries/terraform/general/output_without_description)
- [variable_without_description](https://github.com/Checkmarx/kics/tree/master/assets/queries/terraform/general/variable_without_description)
- [variable_without_type](https://github.com/Checkmarx/kics/tree/master/assets/queries/terraform/general/variable_without_type)

### Steps to Reproduce the Problem

Scanning this code on terraform files works correctly.
```hcl
terraform {
  backend "local" {}

  required_providers {
    aws = {
      source  = "hashicorp/aws"
      version = "~> 4"
    }
  }
  required_version = "~> 1"
}

provider "aws" {
  region = "us-west-2"
}

variable "variable_no_description" {
  type    = string
  default = null
}

variable "variable_no_type" {
  default     = null
  description = "some description"
}

output "output_no_description" {
  value = "test"
}
```
While scanning the generated terraform plan in JSON format don't detect issues.
```json
{
  "format_version": "1.2",
  "terraform_version": "1.5.0",
  "variables": {
    "variable_no_description": { "value": null },
    "variable_no_type": { "value": null }
  },
  "planned_values": {
    "outputs": {
      "output_no_description": {
        "sensitive": false,
        "type": "string",
        "value": "test"
      }
    },
    "root_module": {}
  },
  "output_changes": {
    "output_no_description": {
      "actions": ["create"],
      "before": null,
      "after": "test",
      "after_unknown": false,
      "before_sensitive": false,
      "after_sensitive": false
    }
  },
  "prior_state": {
    "format_version": "1.0",
    "terraform_version": "1.5.0",
    "values": {
      "outputs": {
        "output_no_description": {
          "sensitive": false,
          "value": "test",
          "type": "string"
        }
      },
      "root_module": {}
    }
  },
  "configuration": {
    "provider_config": {
      "aws": {
        "name": "aws",
        "full_name": "registry.terraform.io/hashicorp/aws",
        "version_constraint": "~\u003e 4.0",
        "expressions": { "region": { "constant_value": "us-west-2" } }
      }
    },
    "root_module": {
      "outputs": {
        "output_no_description": { "expression": { "constant_value": "test" } }
      },
      "variables": {
        "variable_no_description": { "default": null },
        "variable_no_type": {
          "default": null,
          "description": "some description"
        }
      }
    }
  },
  "timestamp": "2023-06-16T13:21:33Z"
}

```

Commands used : 
- Terraform plan : `terraform show -json plan-sample.tfplan > plan-sample.tfplan.json` 
- Scanning :  `docker run -t -v "$(pwd)":/src/ checkmarx/kics scan --type Terraform --path /src/plan-sample.tfplan.json`

Using the latest version : `v1.7.1`

**Comment by cx-henriqueAlvelos** [comment]
Hi! There is a [discussion](https://github.com/Checkmarx/kics/discussions/6441) related to this issue. We will look into this problem ASAP.

Best regards
